vercel.json

```
@@ -1,4 +1,15 @@
 {
+  "headers": [
+    {
+      "headers": [
+        {
+          "key": "Content-Type",
+          "value": "text/plain; charset=utf-8"
+        }
+      ],
+      "source": "/"
```

```suggestion
      "source": "/(.*\.py)?"
```

Think something like this should handle both cases.

---

What cases?

---

The file is served on both `/` and `/install-poetry.py`.

```console
$ curl -I https://install-python-poetry-org-git-fork-dimasah-147cc4-python-poetry.vercel.app/install-poetry.py
HTTP/2 200 
date: Tue, 08 Feb 2022 22:35:19 GMT
content-type: application/octet-stream
content-length: 26429
x-robots-tag: noindex
etag: W/"797f9af00b43d53af0d66b20351c89bb547a23337d2672fcfe2c46bdd512f491"
access-control-allow-origin: *
cache-control: public, max-age=0, must-revalidate
content-disposition: inline; filename="install-poetry.py"
accept-ranges: bytes
age: 4244447
x-vercel-cache: HIT
server: Vercel
x-vercel-id: fra1:fra1::2rf6t-1644359719413-b58430301767
strict-transport-security: max-age=63072000; includeSubDomains; preload
```